test/common/vmimages.py

```diff
@@ -38,7 +38,11 @@
 DEVNULL = open("/dev/null", "r+")
 
 CONFIG = "~/.config/image-stores"
-DEFAULT = "https://fedorapeople.org/groups/cockpit/images/"
+DEFAULT = [
+    "https://209.132.184.69:8493/",
```

Comment:
Indeed. In general there are several parts here that needs to move out of the cockpit repo, especially due to the maintenance branches. We need to discuss tactics for this. I have some ideas for specific things, but it would be nice to put those ideas into a complete plan.